tests/torch/model/test_model.py

```diff
@@ -200,6 +234,15 @@ def test_item_prediction_transformer_torch_model_from_config(
     assert out.size()[1] == task.target_dim
     assert len(out.size()) == 2
 
+    in_schema = model.input_schema(max_sequence_length=20)
+    assert isinstance(in_schema, Core_Schema)
+    assert set(in_schema.column_names).issubset(yoochoose_schema.column_names)
```

Comment:
Why is this a subset and not the same?

(In this case `in_schema` is missing `{'ts/list', 'ts/first', 'ts/last', 'timestamp/age_days/list', 'day_idx', 'session_id', 'session_start', 'session_size'}`)

Reply:
I tried this PR in the [Systems integration test with Transformers4Rec](https://github.com/NVIDIA-Merlin/systems/blob/v0.7.0/tests/integration/t4r/test_pytorch_backend.py) and got the following error trying to use the `model.input_schema` to define the input features, suggesting that maybe the missing cols are required?

```
RuntimeError: Number of dimensions of repeat dims can not be smaller than number of dimensions of tensor
```

Reply:
This is a subset of the original schema because the sequential input block in T4Rec (`TabularSequenceFeatures`) filters the inputs based on their tags: It keeps only variables tagged as `['list',  'categorical']` or `['list', 'continuous']`. So the features `'ts/list', 'ts/first', 'ts/last', 'day_idx', 'session_id', 'session_start', 'session_size'` are not lists, and  `'timestamp/age_days/list'` is tagged only as list.
In this unit test, the user can provide the whole schema (generated by NVTabular for example); and the T4Rec model is responsible for filtering the features. 

Can you share the code snippet to reproduce the bug and check what part of T4Rec is causing it? 

Reply:
It turns out that the error was caused by a different issue, not passing the correct shape input for some columns, `[-1, 1]` vs `[-1]` and not because of these missing features